### ISSUE TITLE
Monitoring of predictions on failure cases

### ISSUE THREAD

**Issue by APJansen** [UNSAT3D/unsat#14]
This needs to be rewritten to be compatible with 3D Unet. It should just take the yaml file with the coordinates and dynamically extract the data in the patch size relevant for the model at hand.

**Comment by APJansen** [comment]
This is done in [3689512](https://github.com/UNSAT3D/unsat/commit/36895123b9882832c24fbc1b857acdc3798760f3)

It has two remaining issues:
- while it indicates if a sample was in the training+validation set or the test set, it doesn't distinguish the first two
- the plots won't be centered on the point of interest if either no patches are used during training or if the center is close to the border

For the first point, it may be easiest to change the way the train/val split is done. Currently a selection is made on samples and days for the trail+val set, and then randomized along the heights. Perhaps it's better to do the train/val split on days instead.
